ui/components/SelectSource.tsx

```diff
@@ -25,25 +26,31 @@ export default function SelectSource({
   peerType,
   setPeerType,
 }: SelectSourceProps) {
-  const dbTypes = Object.values(DBType)
+  let dbTypes = Object.values(DBType)
```

Comment:
```suggestion
  const dbTypes = Object.values(DBType)
```
`const` in javascript prevents changing binding, but does not prevent mutation